package/gluon-ssid-changer/luasrc/lib/gluon/config-mode/model/admin/ssid-changer.lua

```diff
@@ -0,0 +1,41 @@
+local uci = require('simple-uci').cursor()
+local util = require 'gluon.util'
+
+local pkg_i18n = i18n 'gluon-ssid-changer'
+
+local f = Form(pkg_i18n.translate('Offline-SSID'))
+
+local s = f:section(Section, nil, pkg_i18n.translate(
+	'Here you can enable to automatically change the SSID to the Offline-SSID '
+  .. 'when the node has no connection to the selected Gateway.'
+))
+
+local enabled = s:option(Flag, 'enabled', pkg_i18n.translate('Enabled'))
+enabled.default = uci:get_bool('ssid-changer', 'settings', 'enabled')
+
+-- local prefix = s:option(Value, 'prefix', pkg_i18n.translate('First part of the Offline SSID'))
+-- prefix:depends(enabled, true)
+-- prefix.datatype = 'maxlength(32)'
+-- prefix.default = uci:get('ssid-changer', 'settings', 'prefix')
+
+function f:write()
+	if enabled.data then
+		uci:section('ssid-changer', 'settings', 'settings', {
+			enabled = '1',
+			-- prefix = prefix.data
+			-- switch_timeframe = switch_timeframe.data or '1440'
+			-- tq_limit_max = tq_limit_max.data or '55'
+			-- first = first.data or '5'
+			-- prefix = prefix.data or 'FF_Offline_'
+			-- suffix = suffix.data or 'nodename'
+			-- tq_limit_min = tq_limit_min.data or '45'
+			-- tq_limit_enabled = tq_limit_enabled.data or '0'
+		})
+	else
+		uci:set('ssid-changer', 'settings', 'enabled', '0')
+	end
+
+	uci:commit('ssid-changer')
```

Comment:
i'm not sure whether commit is correct here, maybe there's a central commit already?

Reply:
I am not sure either. ... are there some experts here sure? ;)

Reply:
just compare with other config mode packages and afterwards try it out.

Reply:
should be correct as each module in the "Advanced Setup" tab takes care of persisting it's state itself.